README.md

```diff
@@ -102,3 +87,6 @@ The filesystem can be instantiated for different use cases based on a variety of
 The `AzureBlobFileSystem` accepts [all of the Async BlobServiceClient arguments](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-python).
 
 By default, write operations create BlockBlobs in Azure, which, once written can not be appended. It is possible to create an AppendBlob using `mode="ab"` when creating and operating on blobs. Currently, AppendBlobs are not available if hierarchical namespaces are enabled.
+
+### Older versions
+ADLS Gen1 filesystem has officially been [retired](https://learn.microsoft.com/en-us/lifecycle/products/azure-data-lake-storage-gen1)). Hence the older versions of this package, which was designed to connect to ADLS Gen1 is obsolete.
```

Comment:
A couple of suggestions on this section:
* It looks like there is an extra `)` that we should remove at the end of the link.
* Maybe we can remove this second sentence and just add on: "and support in `adlfs` is obsolete"? Mainly, the AzureBlobFileSystem has been around for several years so technically these older versions should work just fine if customers were using `az` or `abfs`.